src/security/ErrorDetail.cc

```diff
@@ -714,31 +749,32 @@ Security::ErrorDetail::err_lib_error() const
  \retval 0 for unsupported codes
 */
 size_t
-Security::ErrorDetail::convert(const char *code, const char **value) const
+Security::ErrorDetail::convertErrorCodeToDescription(const char * const code, std::ostream &os) const
```

Comment:
This renaming is rather misleading because this method does not convert ErrorCode (yes, there is such a thing in this context) to its description. This code expands `%code`. I recommend leaving this method name unchanged in this PR.

If (and only if) you insist on renaming, I can suggest:

```suggestion
Security::ErrorDetail::expandPercentCode(const char * const code, std::ostream &os) const
```

or

```suggestion
Security::ErrorDetail::substitutePercentCode(const char * const code, std::ostream &os) const
```

or something like that...

Reply:
I like expandPercentCode; "convert" is very  nondescriptive